lib/motors/messages.json

```diff
@@ -0,0 +1,22 @@
+{
+    "config_messages": [
+        {
+            "example_config_message": {
+                "id": 123,
```

Comment:
is this going to be hex or dec?

Reply:
json doesn't support hex, i think we should use hex so the config messages are identical to those in the manual.  therefore store hex as a string and parse into a value maybe?

Reply:
it is dec at the moment though

Reply:
hm, thats very ugly

dont know what a good solution is, having strings there sounds weird too